### DISCUSSION TITLE
show dots seems to work fine with wholerow plugin

### DISCUSSION THREAD

**Discussion by idreesi** [vakata/jstree#2581] [Q&A] [answered]
Hi first of all thanks for the amazing project,

I need to activate dots with the wholerow plugin, but after checking the code I found that you are intentionally hiding dots for wholerow plugin using `this.hide_dots()`  and on different threads you have mentioned that both plugins are not compatible with each other in current version.

I just need to know what will be the consequences of activating dots again with wholerow plugin? As of now for me it seems working great. I did following:
```
.bind('ready.jstree set_state.jstree', function (e, data) {
				data.instance.show_dots();
			})
```

**Comment by vakata** [comment, marked as answer]
As far as I remember - when hovering or selecting the vertical dots set by parent elements will be hidden behind the hovered/selected state.
